CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [1.0.0](https://github.com/jmeas/redux-simple-resource/releases/tag/1.0.0-alpha1) (3/19/17)
+This shares the same release notes as `1.0.0-alpha1`, and is mostly used to
```

Comment:
I don't think we need to say "mostly" here. This sentence describes **all** that the release does, so we can say that.